core/commands/mount_darwin.go

```diff
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"syscall"
 
-	fuseversion "github.com/jbenet/go-ipfs/Godeps/_workspace/src/github.com/jbenet/go-fuse-version"
+	fuseversion "github.com/jbenet/go-fuse-version"
```

Comment:
*sigh* godep vendoring. i _think_ that `godep get ./...` will:
- only fetch things the user doesnt have (so wont break things for them)
- will still require a `godep update <path>` to change to the version the user has.

why did this rewrite even happen? godep has no business turning that back into a non-vendored thing...